libindy/src/api/anoncreds.rs

```diff
@@ -706,12 +712,15 @@ pub extern fn indy_prover_get_claims_for_proof_req(command_handle: i32,
 ///             "requested_predicate_1_referent": [claim_proof2_referent],
 ///             "requested_predicate_2_referent": [claim_proof3_referent],
 ///         }
-///         "claim_proofs": {
-///             "claim_proof1_referent": [<claim_proof>, issuer_did, schema_seq_no, revoc_reg_seq_no],
-///             "claim_proof2_referent": [<claim_proof>, issuer_did, schema_seq_no, revoc_reg_seq_no],
-///             "claim_proof3_referent": [<claim_proof>, issuer_did, schema_seq_no, revoc_reg_seq_no]
-///         },
-///         "aggregated_proof": <aggregated_proof>
+///         "proof": {
+///             "proofs": {
+///                 "claim_proof1_referent": <claim_proof>,
+///                 "claim_proof2_referent": <claim_proof>,
+///                 "claim_proof3_referent": <claim_proof>
+///             },
+///             "aggregated_proof": <aggregated_proof>
+///         }
+///         "identifiers": [{issuer_did, rev_reg_seq_no, schema_key: {name, version, did}}]
```

Comment:
Each part of the proof may belong to different schemas and issuers. Why did we move this information out of `"claim_proofX_referent"`?

Reply:
Changed identifiers format to
"identifiers": {"claim_proof1_referent":{issuer_did, rev_reg_seq_no, schema_key: {name, version, did}}}